torch/onnx/symbolic_helper.py

```diff
@@ -867,6 +881,40 @@ def _set_training_mode(training_mode):
     global _training_mode
     _training_mode = training_mode
 
+# Symbolic function state.
+class SymbolicFunctionState(object):
+    # Skip check due to cannot import IValue from torch._C
+    params_dict = None  # type: ignore[var-annotated]
```

Comment:
Are these class variables rather than instance variables on purpose?
It's a bit weird to instantiate this object and then access its class variables.

Reply:
changed to instance variables